client/post-editor/editor-ground-control/index.jsx

```diff
@@ -91,10 +93,23 @@ export default React.createClass( {
 			showDateTooltip: false,
 			firstDayOfTheMonth: this.getFirstDayOfTheMonth(),
 			lastDayOfTheMonth: this.getLastDayOfTheMonth(),
-			needsVerification: config( 'email_verification_gate' ) && userUtils.needsVerificationForSite( this.props.site ),
+			needsVerification: config( 'email_verification_gate' ) && this.props.userUtils.needsVerificationForSite( this.props.site ),
 		};
 	},
 
+	componentWillReceiveProps: function( nextProps ) {
+		this.setState( {
```

Comment:
Is there any benefit to only running `setState` if the `nextProps` differ from the current props?

`if ( nextProps.needsVerificationForSite( nextProps.site ) !== this.props.userUtils.needsVerificationForSite( this.props.site ) ) { }`